### PR TITLE
Create files and folders to define directories

### PR DESCRIPTION
**Pull request by elineelorm**
- I replicated the google sps directories but instead of portfolio i named that folder app.
- I also created the .gitignore file however it's still empty
- for the pom.xml file, I just copied the same information as what was in that of the portfolio
- i also created an empty index.html file

### CODE REVIEW

File: app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  <properties>
+    <!-- Java 11 -->
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jetty.version>9.4.38.v20210224</jetty.version>
+
+    <!-- Project-specific properties -->
+    <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
+    <googleCloudProjectId>enuviadenu-sps-summer21</googleCloudProjectId>
```
Comment (sbahaud) [inline app/pom.xml:19]: Change the project-id to the groups project id which is "summer21-sps-6"

**Review by sbahaud — COMMENTED** [review]
Update gitignore with everything that is in the portfolio's gitignore

**Review by sbahaud — CHANGES_REQUESTED** [review]
Please copy gitignore from the portfolio

File: app/pom.xml (hunk 1 above)
Comment (elineelorm) [reply app/pom.xml:19]: done

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**